src/utils/fluent.js

```diff
@@ -41,6 +41,23 @@ async function initFluentBundles () {
 
   await Promise.allSettled(promises)
 
+  // Fluent files in this directory are not currently localized, but might be
+  // moved to `/locales/` later to submit them for localization:
+  const unlocalizedDirname = resolve('../unlocalized/en')
```

Comment:
I added the strings for this feature in an `unlocalized` dir, since this feature is (currently) only available to US users.

Reply:
Removed again, since this will be availably to everyone ( :tada: ).